src/main/java/frc/robot/generic/subsystems/drive/Drive.java

```diff
@@ -320,4 +324,14 @@ public double getMaxLinearSpeedMetersPerSec() {
   public double getMaxAngularSpeedRadPerSec() {
     return maxSpeedMetersPerSec / driveBaseRadius;
   }
+  // Reset ModuleIO to absolute position based on module array
+  // Made this a method for implementation into the SwerveBuilder class (that's also why it returns command not void)
+  public Command resetModulesToAbsolute(){
+    // Added getter as the code was a bit more readable than just manually reseting each instance
+    return Commands.runOnce(() ->{
+      for(Module module : modules) {
+        module.getModuleIO().resetToAbsolute();
+      }
+    }, this);
```

Comment:
```suggestion
    }, this).ignoringDisable(true);
```
This function needs to ignore disabling